Sources/MapLibreSwiftUI/Models/MapCamera/MapViewCamera.swift

```diff
@@ -32,25 +47,76 @@ public struct MapViewCamera {
+                                         pitch: Double? = Defaults.pitch) -> MapViewCamera {
         
+        // Coordinate is ignored when tracking user location. However, pitch and zoom are valid.
         return MapViewCamera(state: .trackingUserLocation,
-                             coordinate: location.coordinate,
+                             coordinate: Defaults.coordinate,
+                             zoom: zoom,
+                             pitch: pitch,
+                             course: Defaults.course,
+                             lastReasonForChange: .programmatic)
+    }
+    
+    /// Enables user location tracking within the MapView.
+    ///
+    /// This feature uses the MLNMapView's userTrackingMode = .follow
+    ///
+    /// - Parameters:
+    ///   - zoom: Set the desired zoom. This is a one time event and the user can manipulate their zoom after unlike pitch.
+    ///   - pitch: Provide a fixed pitch value. The user will not be able to adjust pitch using gestures when this is set. Use nil/default to allow user control.
+    /// - Returns: The MapViewCamera representing the scenario
+    public static func trackUserLocationWithHeading(zoom: Double = Defaults.zoom,
+                                                    pitch: Double? = Defaults.pitch) -> MapViewCamera {
+        
+        // Coordinate is ignored when tracking user location. However, pitch and zoom are valid.
+        return MapViewCamera(state: .trackingUserLocationWithHeading,
+                             coordinate: Defaults.coordinate,
+                             zoom: zoom,
+                             pitch: pitch,
+                             course: Defaults.course,
+                             lastReasonForChange: .programmatic)
+    }
+    
+    /// Enables user location tracking within the MapView.
+    ///
+    /// This feature uses the MLNMapView's userTrackingMode = .follow
+    ///
+    /// - Parameters:
+    ///   - zoom: Set the desired zoom. This is a one time event and the user can manipulate their zoom after unlike pitch.
+    ///   - pitch: Provide a fixed pitch value. The user will not be able to adjust pitch using gestures when this is set. Use nil/default to allow user control.
+    /// - Returns: The MapViewCamera representing the scenario
+    public static func trackUserLocationWithCourse(zoom: Double = Defaults.zoom,
+                                                   pitch: Double? = Defaults.pitch) -> MapViewCamera {
+
+        // Coordinate is ignored when tracking user location. However, pitch and zoom are valid.
+        return MapViewCamera(state: .trackingUserLocationWithCourse,
+                             coordinate: Defaults.coordinate,
                              zoom: zoom,
                              pitch: pitch,
-                             course: location.course)
+                             course: Defaults.course,
+                             lastReasonForChange: .programmatic)
```

Comment:
I think this method never got implemented fully since it's still doing pitch things ;)

(Also note possible rename to heading discussed above)